src/mongocxx/client.cpp

```diff
@@ -13,7 +13,9 @@
 // limitations under the License.
 
 #include <mongocxx/client.hpp>
+#include <mongocxx/exception/error_category.hpp>
 #include <mongocxx/exception/operation.hpp>
+#include <mongocxx/exception/private/mongoc_error.hpp>
```

Comment:
Good point! `/test` and `/config` both have `private` subfolders, so I'm not sure what we want to do. I didn't put much thought into this, I was just thinking, "Okay, it's a private thing related to exceptions. I'll put it here."